doc/quickstart/configure.rst

```diff
@@ -310,17 +310,17 @@ YAML configuration file via
 .. code:: yaml
 
   dask:
-    run: <NAME_OF_CLUSTER>
+    use: <NAME_OF_CLUSTER>
 
 or alternatively with in command line via
 
 .. code:: bash
 
-  esmvaltool run --dask='{"run": "<NAME_OF_CLUSTER>"}' recipe_example.yml
+  esmvaltool run --dask='{"use": "<NAME_OF_CLUSTER>"}' recipe_example.yml
```

Comment:
This syntax is a bit awkward. Do you think it would be possible to use the syntax where you can provide nested dicts using dots, as provided by `dask.config`, i.e. `esmvaltool run --dask.use = <NAME_OF_CLUSTER>`?

Reply:
If not, maybe it would be nice to just add an extra command line argument to specify the profile, e.g. `esmvaltool run --dask-profile local-distributed recipe_python.yml`

Reply:
Unfortunately, using something like `--dask.use` is not possible at the moment. The logic that handles this properly for Dask is (I believe) implemented in `dask.config.set`, which we do not use.

I agree that the existing syntax is a bit clunky, but I really don't want to add another configuration option. This will be really confusing.

How about we try to make the syntax `--dask.use` work in a separate PR? This would also come in handy for other configuration options that use nested dictionaries (which I think we will use more in the future).

Reply:
Sounds good